jaeger/charts/linkerd-jaeger/values.yaml

```diff
@@ -31,6 +31,23 @@ collector:
     version: 0.27.0
     pullPolicy: Always
 
+  resources:
+    cpu:
+      # -- Maximum amount of CPU units that the collector container can use
+      limit:
+      # -- Amount of CPU units that the collector container requests
+      request:
+    memory:
+      # -- Maximum amount of memory that collector container can use
+      limit:
+      # -- Amount of memory that the collector container requests
+      request:
+    ephemeral-storage:
+      # -- Maximum amount of ephemeral storage that the collector container can use
+      limit: ""
+      # -- Amount of ephemeral storage that the collector container requests
+      request: ""
+
```

Comment:
https://github.com/linkerd/linkerd2/blob/main/jaeger/charts/linkerd-jaeger/templates/tracing.yaml#L114-L116

Used here but not present in the `values.yaml`

Reply:
:+1: